output.go

```diff
@@ -178,9 +178,12 @@ func (o *Output) HasDarkBackground() bool {
 	return l < 0.5
 }
 
-// TTY returns the terminal's file descriptor. This may be nil if the output is
+// TTY returns the underlying terminal output. This may be nil if the output is
 // not a terminal.
-func (o Output) TTY() File {
+func (o Output) TTY() io.Writer {
```

Comment:
While this may be compatible in many cases, it's still a bit worrying as an API change. Investigating.

Reply:
also: shouldn't it be `io.ReadWriter`?

Reply:
The writer may be `io.Writer`, `io.ReadWriter`, or `*os.File`